app/assets/stylesheets/user.css.scss

```diff
@@ -2,25 +2,27 @@
     padding-top: 150px;
 }
 
+.form-background {
+  display: block;
```

Comment:
Properties should be ordered background-color, display, height, padding, text-decoration